### ISSUE TITLE
IMPORTANT - local images

### ISSUE THREAD

**Issue by franfranfranfran** [nirvin94/prj-rev-bwfs-tea-cozy#1]
https://github.com/nirvin94/prj-rev-bwfs-tea-cozy/blob/master/tea-cozy/index.html#L10
https://github.com/nirvin94/prj-rev-bwfs-tea-cozy/blob/master/tea-cozy/resources/css/style.css#L44
etc...

it is very important for your resources to be included in your project. links to outside sources can easily become broken if that source decides to change where they are stored or take them down. save your images locally to an 'images' folder inside of resources and then link to them using relative file paths.
more info about relative file paths can be found here - http://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/

**Comment by nirvin94** [comment]
Right. Ok that makes sense! I always do that for personal projects, for some reason I thought with these projects the idea was just link them that way. From now on I'll do it all locally! Practice should always be the way we would do it in the real world anyway i suppose! thanks
